pyirf/perf/irf_maker.py

```diff
@@ -523,6 +561,58 @@ def _make_hdu(cls, hdu_name, t, cols):
         hdu.header.set("EXTNAME", hdu_name)
         return hdu
 
+    @classmethod
+    def _make_aeff_hdu(cls, table_energy, table_theta, aeff):
+        """List of columns"""
+        hdu = fits.BinTableHDU.from_columns(
+            [
+                fits.Column(
+                    name="ETRUE_LO",
+                    format=table_energy["ETRUE_LO"].format,
+                    unit=table_energy["ETRUE_LO"].unit.to_string(),
+                    array=np.atleast_2d(table_energy["ETRUE_LO"]),
+                ),
+                fits.Column(
+                    "ETRUE_HI",
+                    table_energy["ETRUE_HI"].format,
+                    unit=table_energy["ETRUE_HI"].unit.to_string(),
+                    array=np.atleast_2d(table_energy["ETRUE_HI"]),
+                ),
+                fits.Column(
+                    "THETA_LO",
+                    table_theta["THETA_LO"].format,
+                    unit=table_theta["THETA_LO"].unit.to_string(),
+                    array=np.atleast_2d(table_theta["THETA_LO"]),
+                ),
+                fits.Column(
+                    "THETA_HI",
+                    table_theta["THETA_HI"].format,
+                    unit=table_theta["THETA_HI"].unit.to_string(),
+                    array=np.atleast_2d(table_theta["THETA_HI"]),
+                ),
+                fits.Column(
+                    "AEFF",
+                    aeff["AEFF"].format,
+                    unit=aeff["AEFF"].unit.to_string(),
+                    array=np.expand_dims(aeff["AEFF"].data, 0),
+                    dim=str(aeff["AEFF"].data.T.shape),
+                ),
+            ]
+        )
+
+        hdu.header.set(
```

Comment:
You should not be needing to set this yourself, astropy automatically takes care of columns types and shapes.

Why do you set this?